cli/src/pcluster/configure/easyconfig.py

```diff
@@ -110,184 +116,210 @@ def _get_subnets(conn, vpc_id):
     return subnet_options
 
 
-def configure(args):
-    # Check for invalid path (eg. a directory)
-    if args.config_file and os.path.exists(args.config_file) and not os.path.isfile(args.config_file):
-        error("Invalid configuration file path: {0}".format(args.config_file))
-
-    pcluster_config = PclusterConfig(config_file=args.config_file, fail_on_error=False, auto_refresh=False)
+def configure(args):  # noqa: C901
 
-    # FIXME: Overriding HIT config files is currently not supported.
-    if pcluster_config.cluster_model == ClusterModel.HIT:
-        error(
-            "Configuration in file {0} cannot be overwritten. Please specify a different file path".format(
-                pcluster_config.config_file
-            )
-        )
+    config_file_path = args.config_file if args.config_file else default_config_file_path()
+    # Check for invalid path (eg. a directory)
+    if os.path.exists(config_file_path):
+        error(f"A file/folder exists at {config_file_path}. Please specify a different file path")
```

Comment:
Minor: we can write: `{config_file_path} already exists. lease specify a different file path.`